src/CsvTextEditor/Constants.cs

```diff
@@ -48,7 +48,9 @@ public static class Edit
             //public static readonly InputGesture CutInputGesture = new InputGesture(Key.X, ModifierKeys.Control);
 
             public const string Paste = "Edit.Paste";
-            //public static readonly InputGesture PasteInputGesture = new InputGesture(Key.V, ModifierKeys.Control);
+
+            public const string FindReplace = "Edit.FindReplace";
+            //public static readonly InputGesture FindReplaceInputGesture = new InputGesture(Key.F, ModifierKeys.Control);
```

Comment:
it intergrated into our editorControl. We can do it manualy by calling service function in every place we want to.  Personally i don't see any problems with service than shows a tool window (ex. DialogService, UIVisualiationService, WaitDialogservice). After removing function "ShowFindReplaceDialog" from service i need to create a new service that   shows this dialog... also i need to pass a Scope to this service....  the same thing as i do with text editorService... ok, will try to change this behavior